### PR TITLE
Since it wasn't too obvious

### PR DESCRIPTION
**Pull request by zg**
Added a new line that makes it a little bit more clear what command we want to run to prepare for branching.

### CODE REVIEW

**Comment by jlord** [comment]
Awesome! Thank so much! :tada:

**Comment by jlord** [comment]
(will rebuild the guide later today & it will go out then)

**Comment by zg** [comment]
Alright, sounds good. :+1:

**Comment by jlord** [comment]
deployed :ship:

Thanks again!

**Comment by zg** [comment]
GitHub really needs to give you some kind of medal for your awesome emoji usage.  I never see these awesome emojis anywhere on GitHub, it's all about `nerdy code` and [links](https://en.wikipedia.org/wiki/Hyperlink)... :v:

**Comment by jlord** [comment]
Ohh we're big into emoji here :octocat: — my friend and fellow GitHubber @muan has this awesome site: [emoji.muan.co](emoji.muan.co) that I use all the time to find just the right :symbols: :grin:

**Comment by zg** [comment]
Nice!! And they're even retina display friendly, which is even nicer!

File: guide/challenges/branches_arent_just_for_birds.html [hunk 1]
```diff
@@ -69,6 +69,8 @@ <h2 class="challenge-name">7 / 11</h2>
       <code>http://githubusername.github.io/repositoryname</code>
 
       <h2>Step: Create a branch</h2>
+      <p>Let's start by creating a new branch on your fork for your contribution:</p>
+      <p><code>$ git checkout gh-pages</code></p>
```
Comment (muan) [inline guide/challenges/branches_arent_just_for_birds.html]: OH hai just passing by. This should probably be `$ git checkout -b gh-pages` instead.

Comment (zg) [inline guide/challenges/branches_arent_just_for_birds.html]: https://github.com/jlord/git-it/pull/153

**Pull request merged**